src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -216,12 +224,17 @@ private TableInfo buildLiveView(String bigQueryProject, String datasetName, Data
     TableId liveViewId = TableId.of(datasetName, table.getName());
     return TableInfo.of(
         liveViewId,
-        ViewDefinition.of(renderLiveViewSql(bigQueryProject, datasetName, table, null)));
+        ViewDefinition.of(renderLiveViewSql(bigQueryProject, datasetName, table, null, null)));
```

Comment:
It's a private method so this is a minor thing but personally I'd use a method overload for this. To me it makes it easier to see that the last two method parameters have "default" values than finding the call to the method.